### ISSUE TITLE
crash: App crashes after sometime 

### ISSUE THREAD

**Issue by deekoder** [minio/alice#101]
```
 E/AndroidRuntime: FATAL EXCEPTION: AsyncTask #4
                                                                    Process: com.minio.io.alice, PID: 29791
                                                                    java.lang.RuntimeException: An error occurred while executing doInBackground()
                                                                        at android.os.AsyncTask$3.done(AsyncTask.java:309)
                                                                        at java.util.concurrent.FutureTask.finishCompletion(FutureTask.java:354)
                                                                        at java.util.concurrent.FutureTask.setException(FutureTask.java:223)
                                                                        at java.util.concurrent.FutureTask.run(FutureTask.java:242)
                                                                        at android.os.AsyncTask$SerialExecutor$1.run(AsyncTask.java:234)
                                                                        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1113)
                                                                        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:588)
                                                                        at java.lang.Thread.run(Thread.java:818)
                                                                     Caused by: java.lang.NullPointerException: Attempt to get length of null array
                                                                        at okhttp3.RequestBody.create(RequestBody.java:78)
                                                                        at com.minio.io.alice.ServerUploadClient.upload(ServerUploadClient.java:54)
                                                                        at com.minio.io.alice.ServerResponseTask.doInBackground(ServerResponseTask.java:55)
                                                                        at com.minio.io.alice.ServerResponseTask.doInBackground(ServerResponseTask.java:48)
                                                                        at android.os.AsyncTask$2.call(AsyncTask.java:295)
                                                                        at java.util.concurrent.FutureTask.run(FutureTask.java:237)
                                                                        at android.os.AsyncTask$SerialExecutor$1.run(AsyncTask.java:234) 
                                                                        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1113) 
                                                                        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:588) 
                                                                        at java.lang.Thread.run(Thread.java:818) 
03-21 18:19:42.592 29791-30060/com.minio.io.alice I/Process: Sending signal. PID: 29791 SIG: 9
```

**Comment by harshavardhana** [comment]
Fixed by #102

**Comment by harshavardhana** [comment]
Fixed.. and merged.
